lxd/storage/drivers/utils.go

```diff
@@ -836,3 +837,30 @@ func loopDeviceAutoDetach(loopDevPath string) error {
 	_, err := shared.RunCommand("losetup", "--detach", loopDevPath)
 	return err
 }
+
+// wipeBlockHeaders will wipe the first 4MB of a block device.
```

Comment:
Why 4mb out of interest?

Reply:
No real reason, it's been my habit for years.
You normally would need a lot less, but wiping a few MBs tends to help with the few tools that scan for more than one superblock on the likes of ext

Reply:
Thanks.